examples/keras_recipes/subclassing_conv_layers.py

```diff
@@ -29,8 +27,8 @@
 [StandardizedConv2D](https://arxiv.org/abs/1903.10520) as shown below.
 """
 import tensorflow as tf
-import tensorflow.keras as keras
-import keras.layers as layers
+import keras
```

Comment:
This example is TF only, let's set the TF backend at the top

Reply:
Done.